ledger-core/v2/log/adapters/bilog/encoder.go

```diff
@@ -6,6 +6,8 @@
 package bilog
 
 import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/log/logoutput"
```

Comment:
something wrong with imports order

Reply:
Fixed. GoLand losses settings sometimes :(

Reply:
oops, this fix will be in a different PR